ts/theme/variables.ts

```diff
@@ -114,6 +118,19 @@ const customVariables = Object.assign(materialVariables, {
   textColor: "#5C6F82",
   textLinkColor: "#0063CF",
   textLinkWeight: "600" as FontWeight,
+  textBoldWeight: "600",
+  textBoldLineHeight: 18,
```

Comment:
can we calculate these from the base font size?

Reply:
Yes, i fix it.

Reply:
Bold is 700 not 600